src/styles/04-templates/_default.scss

```diff
@@ -3,72 +3,72 @@ $sidebar-width: 25%;
 $sidebar-width-s: calc(25% - calc(4 * #{$space-s}));
 
 .main {
-  @include wrapper;
+    @include wrapper;
```

Comment:
Do we want prettier to ignore .scss files?

Reply:
No, I think prettier editing .scss files is fine.